index.html

```diff
@@ -193,6 +193,59 @@ <h3>Skills</h3>
           <img src="imgs/tech-img-3.png" alt="" />
         </div>
       </section>
+
+      <footer class="contact">
+        <img src="imgs/footer-img.png" alt="" />
+        <div>
+          <h2>Get started</h2>
+          <p>
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+            eiusmod tempor incididunt ut labore et dolore magna.
+          </p>
+        </div>
+
+        <form action="https://formspree.io/f/xnqrodae" method="post">
+          <ul>
+            <li>
+              <label for="">Your name</label>
+              <input
+                type="text"
+                id="name"
+                name="username"
+                maxlength="30"
+                placeholder="Full name"
+                required
+              />
+            </li>
+            <li>
+              <label for="">Your email</label>
+              <input type="email" placeholder="Your email" />
+            </li>
+            <li>
+              <label for=""></label>
+              <textarea name="message" maxlength="500" cols="30" rows="10">
+Enter your message here..</textarea
+              >
+            </li>
+            <li><button>Start collaboration</button></li>
```

Comment:
- [ ] I kindly suggest that you add the different button interactions including hover, active (when it's pressed), shown in the Figma template below:

<img width="279" alt="image" src="https://user-images.githubusercontent.com/52934128/198187731-380b420a-fad9-4bfe-b901-788a7cb9f6b7.png">

the button on the pressed state has the `background: #FFFFFF;`, `color: #1A2236;`, `border-width: 2px` and on hover has the `background: #FFFFFF;`, `color: #1A2236;` `border-width: 1px`

### EXAMPLE:
**for pressed state**
```
button:active {
   background-color:  #FFFFFF;;
   color: #1A2236
   border-width: 2px;
}
```
**for hover state**
```
button:hover {
    background-color: #FFFFFF;;
    color: #1A2236
    border-width: 1px;
}

Reply:
Fixed.